### PR TITLE
LPD-9355 Admin return page

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-9355

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/commerce_return/view.jsp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+CommerceReturnListDisplayContext commerceReturnListDisplayContext = (CommerceReturnListDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+request.setAttribute("view.jsp-portletURL", commerceReturnListDisplayContext.getPortletURL());
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/commerce_return/view.jsp:13]: shouldn't we use WebKeys?

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Just a WebKeys to check, I saw other usage in the code, but just make sure it's still fine before forwarding.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:report:2553665163

**Review by luca-pellizzon — APPROVED** [review]

**Pull request closed without merging**